src/nspawn/nspawn-mount.c

```diff
@@ -678,7 +678,7 @@ int mount_all(const char *dest,
                 }
 
                 o = mount_table[k].options;
-                if (streq_ptr(mount_table[k].type, "tmpfs")) {
+                if (streq_ptr(mount_table[k].type, "tmpfs") || streq_ptr(mount_table[k].type, "mqueue")) {
```

Comment:
This code path does not seem to get executed, /dev/mqueue ends up mislabeled within the container.  Should this code path be executed or is /dev/mqueue mounted in some other place in an nspawn container.

Reply:
hmm, that should be the only place really...